test/spec/features/context-pad/ContextPadProviderSpec.js

```diff
@@ -291,6 +291,14 @@ describe('features - context-pad', function() {
       ]);
     }));
 
+    it('should provide text annotation entries', inject(function() {
+      expectContextPadEntries('TextAnnotation_1', [
```

Comment:
Minor: please adhere to our [unit test style](https://github.com/bpmn-io/bpmn-js/blob/c58af15811f9cc7389a69c2fa1286747b07ca256/.github/CONTRIBUTING.md#adhere-to-the-unit-test-style). 

I prepared it:

```suggestion

    it('should provide Text Annotation entries', inject(function() {

      expectContextPadEntries('TextAnnotation_1', [
```